modules/core/src/main/java/org/apache/ignite/internal/visor/query/VisorQueryCacheDetailMetricsCollectorTask.java

```diff
@@ -37,10 +37,10 @@
 import static org.apache.ignite.internal.processors.cache.GridCacheUtils.isSystemCache;
 
 /**
- * Task to collect cache query metrics.
+ * Collect query detailed metrics from cache context only. For compatibility when node does not have indexing module.
```

Comment:
Lets change javadoc:
This task should be used to collect query metrics when indexing feature not available.